build/rust/Cargo.lock

```diff
@@ -174,9 +206,9 @@ dependencies = [
 
 [[package]]
 name = "blake2b_simd"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
we should not be arbitrarily bumping these deps, please revert any rust dependency updates that aren't strictly necessary for this PR. We already have a big problem with this when resolving the build with upstream and changing all of these is going to make it even worse